crates/kitsune_p2p/kitsune_p2p/tests/common/util.rs

```diff
@@ -9,14 +9,14 @@ pub async fn wait_for_connected(
     to_agent: Arc<KitsuneAgent>,
     space: Arc<KitsuneSpace>,
 ) {
-    tokio::time::timeout(std::time::Duration::from_secs(30), async move {
+    tokio::time::timeout(std::time::Duration::from_secs(32), async move {
         loop {
             match sender
                 .rpc_single(
                     space.clone(),
                     to_agent.clone(),
                     "connection test".as_bytes().to_vec(),
-                    Some(std::time::Duration::from_secs(30).as_millis() as u64),
```

Comment:
Internally the `rpc_single` retries discovering peers so we don't need to retry that part from here - but if the connection fails to establish then we wouldn't retry. This should make a difference I think because it's this function that's failing on CI.